tests/drivers/gpio/gpio_basic_api/boards/sim.overlay

```diff
@@ -0,0 +1,13 @@
+/*
```

Comment:
And this one would be named tests/drivers/gpio/gpio_basic_api/boards/native_posix.overlay